docs/docsite/rst/mazer/install.rst

```diff
@@ -0,0 +1,76 @@
+
+.. _mazer_install:
+
+**********
+Installing
+**********
+
+.. contents:: Topics
+
+
+This topic provides instructions on installing Mazer CLI.
+
+Latest Stable Release
+---------------------
+
+The latest stable release is available on `PyPi <https://pypi.org>`_, and can be installed with ``pip``, as shown below:
+
+.. code-block:: bash
+
+    $ pip install mazer
+
+
+Running from Source
+-------------------
+
+The source code for mazer lives at `https://github.com/ansible/mazer <https://github.com/ansible/mazer>`_, and you can
+run the latest, bleeding edge code by cloning the repo, and running ``setup.py``, as shown below:
+
+.. code-block:: bash
+
+    $ git clone https://github.com/ansible/mazer.git
+    $ cd mazer
+    $ python setup.py install
```

Comment:
Please, _never_ advise users to invoke `setup.py` directly. It misses important things pip does, like registering this thing as installed.
If you really want to advise manual clone + install, use `pip install .` (or `pip install -e .`) depending on the needs.
I don't want us officially advising users to run things breaking their envs.